pkg/social/common.go

```diff
@@ -18,3 +21,30 @@ func isEmailAllowed(email string, allowedDomains []string) bool {
 
 	return valid
 }
+
+func jwtTokenScopes(token string) ([]string, error) {
+	chunks := strings.Split(token, ".")
+	if len(chunks) < 2 {
+		return nil, errors.New("malformed token")
+	}
+
+	rawData := chunks[1]
+	if l := len(rawData) % 4; l > 0 {
+		rawData += strings.Repeat("=", 4-l)
+	}
```

Comment:
I guess it covers all cases, but to be sure I can add an option to skip the validation.